SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LayeredLBStrategy.cpp

```diff
@@ -146,6 +150,21 @@ VirtualWorkerId ULayeredLBStrategy::WhoShouldHaveAuthority(const AActor& Actor)
 	return ReturnedWorkerId;
 }
 
+SpatialGDK::FActorLoadBalancingGroupId ULayeredLBStrategy::GetActorGroupId(const AActor& Actor) const
```

Comment:
This function only does anything in a world that doesn't have any of these load balancing objects so we can assume that in that world there are no child strategies or things to override anything. There will be some map that can tell you what group an actor belongs to and that's basically it. We could probably just take the first couple of lines from this and say that's what the interim definition is.

Reply:
The initial version was the first two lines; I've changed it to what's in right now to allow for nested layered strategies.

Reply:
We've discussed this offline; in the end, nesting of LB strats as it looks right now isn't something that is going to be a part of the Strategy Worker. I've applied the suggestion and added a comment describing the reasoning behind this.